aerogear/mobile-security-service-operator
# featc(unbind): Impl unbinding as an finalizer action of app CR (AEROGEAR-9088)

## Motivation
https://issues.jboss.org/browse/AEROGEAR-9088

## What
* Remove unbind type
* Add impl to unbind app into finalizer of app type
* update the readme

## Verification Steps
 
1. Use this PR and the image `docker.io/cmacedo/mobile-security-service-operator:AEROGEAR-9088`
2. Create all with `make create-all`
3. Check that all is running with success then create a new namespace `on new-project myapptest` 
4. Create app by `make create-app`
5. Check that the app was created in the REST Service
6. Run `make delete-app` and check that the app from the REST and configMap were removed with success when the CR was deleted.
 
## Checklist:


## Progress


## Additional Notes
